Description: Fix taking screenshots while using the Gtk.StatusIcon fallback by providing indicator.hide_it() and indicator.show_it()
Origin: commit, revision id: a.starr.b@gmail.com-20130624205916-grnkchdkk5l37sjn
Author: Andrew Starr-Bochicchio <a.starr.b@gmail.com>
Last-Update: 2013-06-24
X-Bzr-Revision-Id: a.starr.b@gmail.com-20130624205916-grnkchdkk5l37sjn
Bug: https://bugs.launchpad.net/kazam/+bug/1190693

=== modified file 'kazam/frontend/indicator.py'
--- old/kazam/frontend/indicator.py	2013-03-01 02:42:40 +0000
+++ new/kazam/frontend/indicator.py	2013-06-24 20:59:16 +0000
@@ -291,3 +291,9 @@
         def start_recording(self):
             logger.debug("Recording started.")
             self.indicator.set_from_icon_name("kazam-recording")
+
+        def hide_it(self):
+            self.indicator.set_visible(False)
+
+        def show_it(self):
+            self.indicator.set_visible(True)

